core/src/test/java/org/elasticsearch/rest/CorsNotSetIT.java

```diff
@@ -31,7 +32,8 @@
 /**
  *
  */
-public class CorsRegexDefaultIT extends ESIntegTestCase {
+@ClusterScope(scope = ESIntegTestCase.Scope.SUITE, numDataNodes = 1)
```

Comment:
I just figured to be explicit, as thats the default but makes it clearer when its spelled out.  Also, the numDataNodes is selected randomly by default, but I didn't see how that would benefit this test, and the other similar tests don't have this.